### ISSUE TITLE
rl-agents compatible with continuous action spaces

### ISSUE THREAD

**Issue by SHITIANYU-hue** [eleurent/rl-agents#52]
I am wondering is [cross entropy method](https://github.com/eleurent/rl-agents/tree/master/rl_agents/agents/cross_entropy_method) the only one that is compatible with continuous action spaces?

I tried [CEM agent](https://github.com/eleurent/rl-agents/blob/master/rl_agents/agents/cross_entropy_method/cem.py#L8), but I found it runs very slow(Animation update is very slow),  how could I increase the running speed? Thanks😁

**Comment by eleurent** [comment]
Hi, 
Yes unfortunately CEM is the only method implemented that handles continuous actions, due to the fact that my own work rather focuses on discrete actions.
To increase the running speed, the first step would be to use parallel computing, since CEM is very easy to parallelize. That should help if you have many CPUs available.
Another possibility is to use the GPU, see e.g. [this tutorial](https://colab.research.google.com/github/eleurent/highway-env/blob/master/scripts/parking_model_based.ipynb) in which a forward dynamical model is trained in pytorch. The advantage is that this model can then be used to forward 100 trajectories in parallel, in a single GPU forward pass, which is much faster (see the CEM reimplementation at the bottom of the notebook).

Another possibility is to use policy gradients algorithms (DDPG, PPO, etc) that handle continuous actions, from other libraries like stable baselines ([example](https://stable-baselines.readthedocs.io/en/master/guide/examples.html#hindsight-experience-replay-her)).

Finally, I would like to try implementing tree-based planning algorithms with continuous actions (that are more sample-efficient than CEM), like the MCTS with progressive widening or SOPC algorithms, but I wont' have the time in the next few weeks.

**Comment by SHITIANYU-hue** [comment]
Thanks for your reply and valuable suggestions!

**Comment by SHITIANYU-hue** [comment]
Hello, I tried the stable baselines with DDPG algorithm, but I found the agent can't learn a reasonable policy. The agent just drives around in a circle.  here is the code(I just directly use their packages):

![image](https://user-images.githubusercontent.com/38762389/97132969-cb2b4380-171e-11eb-9b16-fede40acc6b3.png)

**Comment by Lalayta** [comment]
> Hello, I tried the stable baselines with DDPG algorithm, but I found the agent can't learn a reasonable policy. The agent just drives around in a circle. here is the code(I just directly use their packages):您好，我尝试了使用 DDPG 算法的稳定基线，但我发现代理无法学习合理的策略。代理只是绕着圈子开车。这是代码（我直接使用他们的包）：
> 
> ![image](https://user-images.githubusercontent.com/38762389/97132969-cb2b4380-171e-11eb-9b16-fede40acc6b3.png)

Hi, I've recently encountered this problem, have you solved it?
